sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -39,22 +39,45 @@ The way to obtain group relationship that will determine which graph api will be
 * **direct**: the default value, get groups that the user is a direct member of. For details, see [list memberOf][graph-api-list-member-of] api.
 * **transitive**: Get groups that the user is a member of, and will also return all groups the user is a nested member of. For details, see [list transitive memberOf][graph-api-list-transitive-member-of] api.
 
-### Authenticate in frontend
-Sends bearer authorization code to backend, in backend a Spring Security filter `AADAuthenticationFilter` validates the Jwt token from Azure AD and save authentication. The Jwt token is also used to acquire a On-Behalf-Of token for Azure AD Graph API so that authenticated user's membership information is available for authorization of access of API resources. 
-Below is a diagram that shows the layers and typical flow for Single Page Application with Spring Boot web API backend that uses the filter for Authentication and Authorization.
-![Single Page Application + Spring Boot Web API + Azure AD](resource/auth-in-frontend-with-aad-filter.png)
+### Web Server
+Based on Azure AD as a Web Server, it uses OAuth2 authorization code flow to authentication, and authorizes resources based on the scopes or roles claim in the access token. 
```

Comment:
Check of scopes has been removed, so please update the text and graph

Reply:
Only in web server-side has removed, right?